simple_history/tests/tests/test_commands.py

```diff
@@ -111,3 +111,12 @@ def test_no_historical(self):
                                     stdout=out)
         self.assertIn(populate_history.Command.NO_REGISTERED_MODELS,
                       out.getvalue())
+
+    def test_excluded_fields(self):
+        poll = models.PollWithExcludeFields.objects.create(
+            question="Will this work?", pub_date=datetime.now())
+        models.PollWithExcludeFields.history.all().delete()
+        management.call_command(self.command_name,
```

Comment:
add `stderr=StringIO()` and `stdout=StringIO()` here so the output isn't printed in the console when running tests